dev-meetings/2025/2025-07-18/README.md

```diff
@@ -1,30 +1,175 @@
+
+*need to make a decision on the tutorial ordering adaptions [#5949](https://github.com/gammapy/gammapy/pull/5949)
+
+RT: Make API parts more visible, too many things in the API-part. Discussing of "key-steps" part, clarification needed, not everything as key-step.
+AS: astro_dark_matter also in recipes. Put it only once? Uncomfortable with name "key-steps", suggests you have to do it first.
+KF answers: You can argue, that one should understand the points beforehand. If looking for having extra section for example, things like clustering, theta_square_plot.py would be moved there. Question: What do we want to change for 2.0, if anything.
+RT: Maybe keep API part for DM, priors, etc and something specific for smaller use-cases.
+Decision regarding astro_dark_matter.py not taken yet. 
+KF: Leaving it in the turotials makes more sence yet, not clear how many people go to the recipes.
+RT: We can not rely on gammapy-recipes, as long as it is not stable.
+KF, RT agrees that specific opinions could be put on the PR.
+RT: Decision taking needed soon.
+
+
+* can we merge? [#5886](https://github.com/gammapy/gammapy/pull/5886/)
+* command line option for tutorial setup -- should we remove from all other tutorials now too? [#5941](https://github.com/gammapy/gammapy/pull/5941/)
+
+### PRs and Timeline:
+RT: Many open PR yet with 2.0-milestone. Timeline? End of July not realistic.
+We need to branch, create 2.0, test it. Who is around?
+->AS,KF,BK, available probably in the coming time.
+Open point, changelog needs to be prepared, AS agrees to do it.
+Citations takes time as well (especially. people from last coding sprint).
+
+* Discussion of PR #5881 "A class for vectorized evaluation of flux uncertainties based on samples". 
+
+RT reports: Inconsistency how evaluation is done e.g. on naima models and ??? models, not uniform yet.
+Evaluate function must be vectorizable, otherwise errorband can not be plotted. Do we need fall-back-mechanism?
+For-loop eval of all the parameters?
+Discussion with AS.
+RT: Main thing that needs to be done: Introduce fall back mechanism.
+Regarding whole PR, feature really needed, PR should go in for 2.0.
+
+* Brief discussion of PR #5838 "Addition of FitStatistic penalties"
+
+RT: Can come later.
+
+* Brief discussion of PR #5815
+
+ RT: Have to talk with BK.
+
+
+RT: Regarding code, do we agree to postpone to mid august / Aug 20? 
+AS agrees, end of August should be hard deadline.
+RT: Branching maybe in two weeks.
+
+* Brief discussion of PR #5910 with KF.
+
+  KF: First changed only name of params, but tests fails. What is final verdict?
+  RT: Have to check how it is used in the code.
+
+* Brief discussion of PR #5960. 
+
+AS: Discussed last week. Issue that edges are in true_energy, but cut in recon energies (???).
+KF agrees. How to do it?
+RT: Not go in direction of weights, rather tresholds. It would otherwise change the logic. You can try to see, if it is possible, but would be different mechanism.
```

Comment:
```suggestion
RT: Not go in direction of weights, rather hard thresholds. It would otherwise change the logic. You can also try to see if it is possible to filter on reco exposure instead, but would be different mechanism.
```